Sources/Conformance/main.swift

```diff
@@ -84,7 +84,7 @@ func buildResponse(serializedBytes: [UInt8]) -> Conformance_ConformanceResponse
         return response
     }
 
-    let msgType: SwiftProtobuf.Message.Type
+    let msgType: any SwiftProtobuf.Message.Type
```

Comment:
I believe this is wrong.

Reply:
I think this _is_ an existential use of `Message.Type` since the type assignment is dynamic below. `msgType` could be assigned to `ProtobufTestMessages_Proto3_TestAllTypesProto3` or `ProtobufTestMessages_Proto2_TestAllTypesProto2` so the concrete type implementing the `Message` protocol is not known.